packages/ui/src/components/ProjectCard/ProjectCard.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { ProjectCard } from './ProjectCard';
+
+type StatusArray = {
+  status: 'idle' | 'doing' | 'review' | 'done';
+  text: string;
+}[];
+
+const title = 'Zaawansowany React i Node';
+const date = new Date();
+const statusArray: StatusArray = [
+  { status: 'idle', text: 'Nie rozpoczęty' },
+  { status: 'doing', text: 'W trakcie' },
+  { status: 'review', text: 'W ocenie' },
+  { status: 'done', text: 'Zakończony' },
+];
+
+describe('ProjectCard', () => {
+  statusArray.map((el) => {
```

Comment:
💭 I think, creating four separate tests instead of array mapping can be easier to read as there would be no conditional logic to track in the test.